packages/notification-services-controller/notification-services/mocks/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@metamask/notification-services-controller",
   "version": "1.0.0",
+  "private": true,
```

Comment:
I've added this to explicitly set these internal subpath exports to be private.
This shouldn't cause issues with publishing as these are not specified on root of the workspaces file.

If this does cause issues with publishing, I can revert.